xml/deploy_cephadm.xml

```diff
@@ -441,8 +442,9 @@ o- minions ................................................. [Minions: 5]
    <sect3 xml:id="deploy-cephadm-configure-cephadm">
     <title>Specify &sminion;s Managed by &cephadm;</title>
     <para>
-     Specify which minions will belong to the &ceph; cluster and will be
-     managed by &cephadm;:
+     Specify which hosts will belong to the &ceph; cluster and will be managed
+     by &cephadm;. Include all hosts that will run &ceph; services as well as
+     the &adm;:
```

Comment:
`as well as the &adm;` -> Just for reference, this will likely change with https://github.com/ceph/ceph-salt/issues/252 but for now, that's correct and we should keep this sentece.